aiida/cmdline/params/options/__init__.py

```diff
@@ -391,3 +391,17 @@ def active_process_states():
 
 CONFIG_FILE = ConfigFileOption(
     '--config', help='Load option values from configuration file in yaml format.')
+
+DICT_FORMAT = OverridableOption(
+    '-f', '--format', 'fmt',
+    type=click.Choice(["json+date", "yaml"]),
+    default="json+date",
+    help="format for the output"
```

Comment:
Please use a proper sentence for the help string just like all the other options, i.e. start with a capital and end with a full stop